modules/nw-ptp-configuring-linuxptp-services-as-boundary-clock.adoc

```diff
@@ -31,133 +31,7 @@ See "Configuring the PTP fast event notifications publisher" for more informatio
 .Recommended PTP boundary clock configuration
 [source,yaml]
 ----
----
```

Comment:
These are complete YAML configs, so the following guidelines do not apply?

![Screenshot from 2023-09-27 12-40-48](https://github.com/openshift/openshift-docs/assets/57954076/a8672f91-0339-4ed2-b525-ac2035429b1a)

Reply:
https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html#yaml-basics